### PR TITLE
php@7.4 7.4.13 (new formula)

### PR DESCRIPTION
**Pull request by derrabus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Today, the release of php 8.0.0 ~~is scheduled~~ happened. This PR proposes to introduce a versioned formula for php's 7.4 branch which will receive bugfixes until Nov 2021 and security fixes until Nov 2022, according to https://www.php.net/supported-versions

### CODE REVIEW

**Comment by derrabus** [comment]
#65698 needs to be merged first, apparently.

File: Aliases/php@7.4 [hunk 1]
```diff
@@ -1 +0,0 @@
-../Formula/php.rb
```
Comment (SMillerDev) [inline Aliases/php@7.4:1]: Please don't remove this file (yet)

**Review by SMillerDev — COMMENTED** [review]
This really needs to _just_ be a version update

File: Formula/php@7.4.rb [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+  url "https://www.php.net/distributions/php-7.4.13.tar.xz"
+  mirror "https://fossies.org/linux/www/php-7.4.13.tar.xz"
+  sha256 "aead303e3abac23106529560547baebbedba0bb2943b91d5aa08fff1f41680f4"
+  license "PHP-3.01"
+
+  bottle do
+    sha256 "38d2273cc27903564dc848ebc44a97c1d896f3a14d2cdaf4af207a0972f12206" => :big_sur
+    sha256 "0a0dfe81f00955ecbaf00db3207182bbdb6e6718fca4049f8cfd780a86013494" => :catalina
+    sha256 "3a255c83522169aa287769d59e156abd585f3033cf6641cd6486863fe8c19dc3" => :mojave
+  end
```
Comment (SMillerDev) [inline Formula/php@7.4.rb:14]: no need to add a bottle block

**Comment by LeRoyJulian** [comment]
This need to be merged urgently as PHP 8.0 is now the latest release available, it's not currently possible to downgrade or specifically install PHP 7.4.

**Comment by SudoGetBeer** [comment]
> This needs to be merged urgently as PHP 8.0 is now the latest release available, it's not currently possible to downgrade or specifically install PHP 7.4.

If you have not deleted the old formula you can downgrade with "brew switch php 7.4.12" (you can get the list of old versions with brew list --versions php.
Then you must link back to php (brew link php)

**Comment by SMillerDev** [comment]
> This needs to be merged urgently as PHP 8.0 is now the latest release available, it's not currently possible to downgrade or specifically install PHP 7.4.

You can use `brew extract` if you want to keep a specific version of a formula. Everyone is volunteering their time here so it'll be merged when it's ready and someone has time to review/fix it.

**Comment by derrabus** [comment]
CI is green. 🚀 🐘

**Review by stanislav-janu — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for another contribution! Without people like you submitting PRs we couldn't run this project. You rock, @derrabus!

**Pull request closed without merging**